keep-ui/app/incidents/incident.tsx

```diff
@@ -104,24 +128,24 @@ export default function Incident() {
                 </Button>
               </div>
             </div>
+
```

Comment:
I propose to move filters from the card to make it less nested, more clean:
```suggestion
<IncidentTableFilters />
```

<img width="1936" alt="Screenshot 2024-10-04 at 12 31 48" src="https://github.com/user-attachments/assets/dca48740-d2bf-41d9-b183-aa147cbb64d6">
<img width="616" alt="Screenshot 2024-10-04 at 12 32 53" src="https://github.com/user-attachments/assets/36731d62-e792-434c-83f1-77ad9a6a415b">